activity_hub.php

```diff
@@ -380,19 +376,17 @@ function summarize_stage($stage, $desired_states, $show_filtered_projects=FALSE,
 
     // Every row gets a label, name, and access information.
     echo "<td class='stage-column' $span_rows>";
-    echo "<div class='stage-icon'>$stage_id_bit</div>";
 
     // Output the access status icon. If the user does not yet have access
     // make the image a link to the access requirements.
-    echo "<div class='stage-access'>";
+    echo "<div class='stage-access $access_class'>";
```

Comment:
Should there be a title with the access text in here?

Reply:
Yes indeed - thank you for catching that!